bip-0155.mediawiki

```diff
@@ -78,8 +78,8 @@ This means that the message contains a serialized <code>std::vector</code> of th
 
 One message can contain up to 1,000 addresses. Clients SHOULD reject messages with more addresses.
 
-Field <code>addr</code> has a variable length, with a maximum of 32 bytes (256 bits). Clients SHOULD reject
-longer addresses.
+Field <code>addr</code> has a variable length, with a maximum of 64 bytes (512 bits).
```

Comment:
I think that Wladimir actually meant 512 bytes, it may seems large but it's a DoS limit.

Reply:
Oh, yes, that comment clearly says 512 **bytes**. Updated.